src/engraving/compat/midi/midiinstrument.cpp

```diff
@@ -424,7 +424,7 @@ MidiInstrument minstr[] = {
     { 4, 0, 70, 98, 0, "AirBells" },
     { 4, 0, 71, 98, 0, "BellHarp" },
     { 4, 0, 72, 98, 0, "Gamelmba" },
-    { 7, 0, 0, 99, 0, "Athmosphere" },
+    { 7, 0, 0, 99, 0, "Atmosphere" },
```

Comment:
I really don't feel confident about this. Would you mind dropping the three other changes to this file too?